changelog/6296.improvement.md

```diff
@@ -0,0 +1,10 @@
+* Modified functionality of ``SingleStateFeaturizer`` 

```

Comment:
also add that `Binary...` one is deprecated. and the difference is controlled by split intents in nlu tokenizer